stdeb.cfg

```diff
@@ -1,5 +1,5 @@
 [DEFAULT]
 No-Python2:
-Depends3: python3-setuptools, python3-yaml
+Depends3: python3-flake8, python3-pydocstyle, python3-setuptools, python3-yaml
```

Comment:
No idea if these addition is appropriate. I just added because I got error saying that they are missing when running `nosetests3`.

Reply:
I see for CI the test dependency is installed in the [CI config (.github/workflows/ci.yml#L34)](https://github.com/dirk-thomas/vcstool/blob/2e90383a09e810049ee505e1e618690957c631dc/.github/workflows/ci.yml#L34). So taking down the change on this line.

Running `pytest` locally was inevitable part (otherwise I'd have pushed commit every change during experimenting). Wish test dependency is defined not just in CI config.